src/types/form_feedback.ts

```diff
@@ -8,6 +8,7 @@ export interface IFormFeedback {
   comment: string
   created: Date
   lastModified?: Date
+  _id: any
```

Comment:
nit: A tighter typing (but is still `any` to be honest....) is to use `mongoose.Document`'s `_id` typing.

```suggestion
  _id: Document['_id']
```

Up to you if you want haha